src/luascript.cpp

```diff
@@ -15349,6 +15351,32 @@ int LuaScriptInterface::luaTalkactionSeparator(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaTalkactionAccess(lua_State* L)
+{
+	// talkAction:access(needAccess = false)
```

Comment:
to be really part of *Revscripts*, you should be able to get and set a value with the same function, also the parameters should have a cleaner name

example:
```c++
int LuaScriptInterface::luaTalkactionAccess(lua_State* L)
{
	// get: talkAction:access() or set: talkAction:access(bool)
	TalkAction* talk = getUserdata<TalkAction>(L, 1);
	if (talk) {
		if (lua_gettop(L) == 1) {
			pushBoolean(L, talk->getAccess());
		} else {
			talk->setAccess(getBoolean(L, 2));
			pushBoolean(L, true);
		}
	}
	else {
		lua_pushnil(L);
	}
	return 1;
}
```

Reply:
I think evil hero was very clear with it 
![image](https://user-images.githubusercontent.com/28090948/108914287-7e3e7780-7601-11eb-93e2-11fb3cd61c53.png)

Reply:
Evil Hero mentioend that there shouldn't be `get` methods because revscriptsys wasn't designed for that purpose but only to register scripts. 🤔

Reply:
You're right, this is only a feature for custom servers.
the small example of use would be the following:
```lua
local talk = TalkAction("/comandos", "!comandos", "/commands", "!commands")

function talk.onSay(player, words, param)
	local description = ""
	for words, talk in pairs(Game.getTalkActions()) do
		if not talk:access() or player:getGroup():getAccess() then
			if description == "" then
				description = description .. '\n'
			end
			description = description .. words
		end
	end
	player:showTextDialog(26654, description)
	return false
end

talk:register()
```